app/assets/javascripts/graph.js

```diff
@@ -0,0 +1,98 @@
+var StateId = document.getElementById("circle_pack").getAttribute('data-state-id');
+
+$.ajax({
+         type: "GET",
+         contentType: "application/json; charset=utf-8",
+         url: '/states/' + StateId,
+         dataType: 'json',
+         success: function (data) {
+             draw(data);
+         },
+         error: function (result) {
+             error();
+         }
+ });
+
+function draw(data) {
+  var $circle_pack = $('#circle_pack');
+  var StateId = $circle_pack.data('state-id');
```

Comment:
Identifier '$circle_pack' is not in camel case.<br>Mixed double and single quotes.